interp/run.go

```diff
@@ -1039,22 +1039,33 @@ func genInterfaceWrapper(n *node, typ reflect.Type) func(*frame) reflect.Value {
 				vv = v.Elem()
```

Comment:
vv is unused in the following. So do we not care anymore about this reflect.Ptr case?